src/test/java/util/xcmailr/AbstractXcmailrApiTest.java

```diff
@@ -0,0 +1,63 @@
+package util.xcmailr;
+
+import java.io.IOException;
+import java.util.Base64;
+import java.util.UUID;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.apache.http.client.ClientProtocolException;
+import org.junit.After;
+import org.junit.BeforeClass;
+
+import util.xcmailr.util.Credentials;
+import util.xcmailr.util.SendRequest;
+
+public class AbstractXcmailrApiTest extends AbstractTest
+{
+    protected static final Credentials CREDENTIALS = ConfigFactory.create(Credentials.class, System.getenv());
+
+    protected static final String validMinutes = "1";
+
+    protected static String tempEmail;
```

Comment:
tempEmail should not be static. It should also not be set in a `BeforeClass` block as we want it to differ for each test, so it should be in a `Before` block.